python/lib/dcoscli/dcoscli/data/help/job.txt

```diff
@@ -10,7 +10,7 @@ Description:
         dcos job remove <job-id> [--stop-current-job-runs]
         dcos job show <job-id>
         dcos job update <job-file>
-        dcos job kill <job-id> [<run-id>][--all]
+        dcos job kill <job-id> (<run-id>|--all)
```

Comment:
should we maybe also improve the message when someone calls it with just jobid? So that it does not spit out this 'NoneType' thing but says something meaningful?

Reply:
By default it would print an `Invalid subcommand usage` error message with the help menu below. If you want a specific message this should be a check at the beginning of the command instead.

Reply:
oh ok, so this file is used also for validation, cool ... then it should be enough, thanks!